server/data/helpers/teacherHelper.js

```diff
@@ -1,38 +1,39 @@
 const db = require('../../config/dbConfig');
 
 module.exports = {
-	getTeachers: async (id) => {
+	getAll: async (id) => {
```

Comment:
Does the id need to be passed if this gets all teachers?

Reply:
Good catch!